module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -267,6 +277,42 @@ public function supportsConnectingLibraryCard($authMethod = null)
             && $this->getAuth($authMethod)->supportsConnectingLibraryCard();
     }
 
+    /**
+     * Is persistent login supported?
+     *
+     * @param User $user optional; check user auth method in database
+     *
+     * @return bool
+     */
+    public function supportsPersistentLogin($user = null)
+    {
+        if (!empty($this->config->Authentication->persistent_login)) {
+            if ($user) {
```

Comment:
The problem happens when logging in with a token, basically this:
Log in and save the login -> delete session -> go to /MyResearch/Profile
The logging in works, but saved logins are not displayed because auth method is not updated. Looking at it now, I think we should probably make sure to update auth method when logging in with a token. And if that is the case, where should the info about the auth method come from? From database, or should the login token cookie carry auth method info?

Reply:
I think it probably makes sense to include that information in the database -- if it's carried by the cookie, isn't it easier for the user to manipulate, or do I misunderstand what you mean?

Reply:
@demiankatz  Now the auth method is updated when the user logs in with a cookie. Auth method is checked from the database. I also changed `makeAuth()` function to make the comparison between user auth method and legal auth methods case insensitive. This is because auth methods are stored lowercase in the database.